<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_requirement</class>
 <widget class="QWidget" name="form_requirement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>39</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hlayout_code">
     <item>
      <widget class="QLabel" name="lbl_code">
       <property name="text">
        <string>Code*:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_code"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_systems">
     <item>
      <widget class="QLabel" name="lbl_systems">
       <property name="text">
        <string>System(s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lw_systems">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_sections">
     <item>
      <widget class="QLabel" name="lbl_sections">
       <property name="text">
        <string>Section(s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lw_sections">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_definition">
     <item>
      <widget class="QLabel" name="lbl_definition">
       <property name="text">
        <string>Definition*:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_definition"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_requirement">
     <property name="leftMargin">
      <number>300</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
